packages/docs/docs/getting-started/faqs.md

```diff
@@ -0,0 +1,28 @@
+---
+
+title: FAQs
+---
+
+# ❓FAQs
+
+What is Unirep?
+
+* UniRep is an anonymous reputation protocol. The protocol allows developers to build applications that associate data with anonymous users. 
+
+What is an epoch key?
+
+* Epoch Keys are how a user stays anonymous while receiving data. An epoch key is a temporary anonymous identity or persona that a user uses to interact with a Unirep application across epochs that are renewed each epoch.
```

Comment:
you can give an example that epoch keys are like a set of ethereum wallet addresses, you can receive data from others, but others cannot link the address to one user. and also Unirep can prove the total assets (from all addresses) of one user